bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -586,24 +593,34 @@ def _str_or_none(value):
         return str(value)
 
 
+def _split_id(full_id):
```

Comment:
The docstring is missing here, let's add it to this helper, too.

Reply:
Will it be understandable if I use `Helper: split full_id into composite parts` as the docstring? Or should I write more specific?

Reply:
One liner is fine IMO. Personally, I would also document the argument types and the return value on top of it.

Reply:
@plamut I decided to follow your advice and practice the docstrings writing. Could you have an eye here and check it, please?

![code_sample_2](https://user-images.githubusercontent.com/46078689/64341811-d5a5ed80-cff1-11e9-8b64-2346db14da14.png)

Reply:
@Emar-Kar The docstring looks good, thanks!